src/tools/concurrency/ActorExecutionTrace.java

```diff
@@ -42,13 +43,15 @@
 public class ActorExecutionTrace {
 
   private static final int BUFFER_POOL_SIZE = Runtime.getRuntime().availableProcessors() * 4;
-  private static final int BUFFER_SIZE = 4096 * 1024;
+  static final int BUFFER_SIZE = 4096 * 1024;
   private static final byte MESSAGE_BASE = (byte) 0x80;
   private static final byte PROMISE_BIT = 0x40;
   private static final byte TIMESTAMP_BIT = 0x20;
   private static final byte PARAMETER_BIT = 0x10;
-  private static final int MESSAGE_SIZE = 45;
+  private static final int MESSAGE_SIZE = 45 + 16;
```

Comment:
What's the size based on? Where do the magic numbers come from?
Would be good to have that clear in the code, so that it is obvious when one needs to change this number.